packages/eslint-config-base/rules/imports.js

```diff
@@ -93,6 +93,14 @@ module.exports = {
         ],
       },
     ],
+    /**
+     * Use named exports over default exports
+     * @link https://github.com/reside-eng/guidelines/blob/main/rfcs/shared-patterns/typescript/001-consistent-import-export.md
```

Comment:
It depends on what you do with the comments. Some JSDoc parsers will output a specific type of documentation structure depending on whether you use `@see` or `@link`. There are a lot of ways to use both. 

Since the only (present) use case for these comments is for human readers within GitHub or an IDE, I went with `@link` since it is easy to read. In truth, neither of these JSDoc handles are really needed since IDEs will figure out that this is a link, but I went with `@link` for legibility.

Reply:
@prescottprue is this use of `@link` acceptable, or do you want me to update it per your suggestion?

Reply:
I think this is fine. Going to resolve. @prescottprue reopen if you feel differently.